inbox/api/views.py

```diff
@@ -0,0 +1,70 @@
+# =================================================================================================
+#                                  All Rights Reserved.
+# =================================================================================================
+# File description:
+#       In other frameworks you may also find conceptually similar implementations named
+#       something like 'Resources' or 'Controllers'.
+#
+#       Ref: https://www.django-rest-framework.org/api-guide/viewsets/
+#
+# =================================================================================================
+#    Date      Name                    Description of Change
+# 13-Mar-2022  Wayne Shih              Initial create
+# $HISTORY$
+# =================================================================================================
+
+
+from notifications.models import Notification
+from rest_framework import viewsets, status
+from rest_framework.decorators import action
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from inbox.api.serializers import NotificationSerializer, DefaultNotificationSerializer
+
+
+class NotificationViewSet(viewsets.GenericViewSet):
+    queryset = Notification.objects.all()
```

Comment:
you can also override get_queryset(self) to only return Notification.objects.filter(recipient=self.request.user)

Reply:
I will think about the other use cases and consider this suggestion.
If so, I will make the change with the next notification PR together.

Reply:
If you do override get_queryset(), consider your use case, you could try to make
filter(), order_by(), and prefetch_related() all in the get_queryset()